lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1884,7 +1884,7 @@ sub investigate ($self, %args) {
               // undef
         } ($prev, $self);
         $inv{diff_packages_to_last_good} = $self->packages_diff($prev, $ignore) // 'Diff of packages not available';
-        last unless $prev_file;
+        last unless ($self_file and $prev_file);
```

Comment:
Why not just
```suggestion
        last unless $self_file && $prev_file;
```

Reply:
You say "just". Is there a difference? Maybe this is something tidy should check

Reply:
The operators can make a difference but here we don't so I suggest to stick to our default of using `%%` unless we're chaining commands and avoid braces in simple cases. Maybe tidy can help us with the braces but likely not with the operator.

Reply:
Changed it. I'm used to using `and` and `or` because I find it more readable.